src/components/TextLink/__tests__/TextLink.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import { propTests } from '../../../../tests/playwright';
+import { TextLinkForTest } from './TextLink.story';
+
+test.describe('TextLink', () => {
+  test.describe('visual', () => {
+    [
+      ...propTests.labelPropTest,
+    ].forEach(({
+      name,
+      onBeforeTest,
+      props,
+    }) => {
+      test(name, async ({
+        mount,
+        page,
+      }) => {
+        if (onBeforeTest) {
+          await onBeforeTest(page);
+        }
+
+        const component = await mount(
+          <TextLinkForTest
+            {...props}
+          />,
+        );
+
+        const screenshot = await component.screenshot();
+        expect(screenshot).toMatchSnapshot();
+      });
+    });
+  });
+
+  test.describe('non-visual', () => {
+    test('href', async ({ mount }) => {
+      const component = await mount(
+        <TextLinkForTest />,
```

Comment:
Story should contain all required values so it renders without error. But for purpose of the test, it might be better to explictely write tested prop here so it does not looks like a magic.

variants:

1. Use the same value as it is in story
2. Use another value

@lukasbriza agrees that is would be more explicit. What do you think, @mbohal?

Reply:
I think 2) is the correct solution.